Makefile

```diff
@@ -47,6 +47,7 @@ SUMMON ?= summon --up -p gopass -e service-stac-$(ENV)
 GIT_HASH := `git rev-parse HEAD`
 GIT_BRANCH := `git symbolic-ref HEAD --short 2>/dev/null`
 GIT_DIRTY := `git status --porcelain`
+GIT_TAG := `git describe --tags 2>/dev/null || echo 'unknown'`
```

Comment:
I would only use annotated tag ! I would also avoid writing stderr to null, if an error occur we should have a dirty tag and not a correct  one. Moreover the `git describe` command adds info to the tag if the current commit is not on the tag see https://git-scm.com/docs/git-describe. We should have a bullet proof method of getting the tag to avoid making image with wrong tag.

Reply:
What's your reasoning for "only using annotated tag"?

Reply:
And in what way are annotated tags different to leightweight tags when triggering build based on push to branch and not based on creation of tags?

Reply:
and btw: the `2>/dev/null` stems from two lines above... plus if you're in local development I think it's ok not to have a tag yet

Reply:
From https://git-scm.com/docs/git-tag 

> Annotated tags are meant for release while lightweight tags are meant for private or temporary object labels. For this
reason, some git commands for naming objects (like git describe) will ignore lightweight tags by default.

That's mainly why I would only used annotated tag. Also annotated tags contains meta data like message, creation date, author while lightweight tag not.

Reply:
> and btw: the `2>/dev/null` stems from two lines above... plus if you're in local development I think it's ok not to have a tag yet

I agree that for local development it is ok not to have a tag, but `git describe` will return a previous tag if the current commit is not tagged ! We should not mark an image with a wrong tag.

Reply:
OK I've just tried the `git describe` command from your branch (which contains already a tag) and it gave `v0.2.0-beta.1-2-g17f5fcc` which is fine for a dirty tag. So I suggest simply to remove the `--tags` option and to use annotated tag

Reply:
Interestingly, github doesn't treat leightweight tags as being very 'private' or 'temporary', they show up in tags list as any other tag (in fact when you create a tag via UI it's a leightweight tag). And for `develop` one could even argue that a leightweight tag is the right thing, since it's more of 'private' and 'temporary' character than a release.